.eslintrc.js

```diff
@@ -3,8 +3,14 @@
 require('@uniswap/eslint-config/load')
 
 module.exports = {
-  extends: '@uniswap/eslint-config/react',
+  extends: ['@uniswap/eslint-config/react'],
```

Comment:
I can back out this array-ifying change - it's a no-op from a separate thing I was experimenting with locally.